node/src/service.rs

```diff
@@ -275,10 +276,7 @@ where
 		hwbench.clone(),
 	)
 	.await
-	.map_err(|e| match e {
-		RelayChainError::ServiceError(polkadot_service::Error::Sub(x)) => x,
-		s => s.to_string().into(),
-	})?;
+	.map_err(|e| sc_service::Error::Application(Box::new(e)))?;
```

Comment:
Removal of `RelayChainError`, followed from [Astar's](https://github.com/AstarNetwork/Astar/blob/b4ea96abd7054e86d2132d7e63a9415632fc4a0d/bin/collator/src/parachain/service.rs#L722)